<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Bhargavi" name="Regression suite 1(failed)" tests="4" failures="2" timestamp="16 Jun 2023 14:58:45 GMT" time="0.07" errors="0">
  <testcase name="loginTest" time="0.018" classname="testNGDemo.TestNGDemo4"/>
  <testcase name="customerDetails" time="0.001" classname="testNGDemo.TestNGDemo4">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at testNGDemo.TestNGDemo4.customerDetails(TestNGDemo4.java:15)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- customerDetails -->
  <testcase name="funC" time="0.003" classname="testNGDemo.TestNGDemo3">
    <skipped/>
  </testcase> <!-- funC -->
  <testcase name="funB" time="0.001" classname="testNGDemo.TestNGDemo3">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at testNGDemo.TestNGDemo3.funB(TestNGDemo3.java:16)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- funB -->
</testsuite> <!-- Regression suite 1(failed) -->
